docs/source/cli_help/status.rst

```diff
@@ -87,3 +88,5 @@ With ``--color=always``, git machete always emits colors and with ``--color=auto
 
 ``machete.status.extraSpaceBeforeBranchName``
     .. include:: status_config_key.rst
+        :start-line: 2
+        :end-line: 6
```

Comment:
Huh okay... this is is to skip the part including and after `For example, by default the status is displayed as:`? 🤔

Reply:
Yes

Reply:
`start-line` is to skip `note` formatting, `end-line` to skip `For example, by default the status is displayed as`

Reply:
Okay, I've rendered it locally, makes sense indeed! Pretty brittle since the line numbers can change huh... is it possible to make Sphinx `include` just the given paragraph/section identified by name? or maybe... at least add a comment in status_config_key.rst that the line numbering is relevant